roles/openshift_facts/defaults/main.yml

```diff
@@ -59,6 +59,7 @@ openshift_master_default_subdomain: "{{ osm_default_subdomain }}"
 
 openshift_portal_net: "{{ openshift_master_portal_net | default(None) }}"
 openshift_cluster_network_cidr: "{{ osm_cluster_network_cidr | default('10.128.0.0/14') }}"
+openshift_cluster_networks: "{{ osm_cluster_networks | default([]) }}"
```

Comment:
I am proposing the value for osm_cluster_networks and openshift_cluster_networks should reflect the structure of clusterNetworks  in master-config.yaml and so be a list of dictionaries:
```
osm_cluster_networks:
- cidr: 172.64.0.0/16
  host_subnet_length: 8
- cidr: 172.65.0.0/16
  host_subnet_length: 8
```
I'm not sure how this would look as a dictionary. Can you explain what you are proposing?

Reply:
You're right, my mistake.